src/test/scala/encry/settings/AdditionalTestSettings.scala

```diff
@@ -0,0 +1,6 @@
+package encry.settings
+
+trait AdditionalTestSettings {
+  //settings with known peers
+  lazy val additionalTestSettings: EncryAppSettings = EncryAppSettings.loadConfig("AdditionalTestSettings.conf")
```

Comment:
I don't think that it's necessary to create another config file, because of we want to change only one param such as known peers. We can still use MainTestSettings in tests and use .copy()